configs/sites/discover/packages.yaml

```diff
@@ -101,10 +101,10 @@ packages:
     externals:
     - spec: findutils@4.5.12
       prefix: /usr
-  flex:
-    externals:
-    - spec: flex@2.5.37+lex
-      prefix: /usr
+  #flex:
```

Comment:
Include the `# Don't use, leads to duplicate packages being built` comment warning about duplicate packages?